src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -1094,7 +1132,7 @@ private function renderEditProductForm(FormInterface $productForm, int $productI
             'editable' => $this->isGranted(Permission::UPDATE, self::PRODUCT_CONTROLLER_PERMISSION),
             'taxEnabled' => (bool) $configuration->get('PS_TAX'),
             'stockEnabled' => (bool) $configuration->get('PS_STOCK_MANAGEMENT'),
-            'isMultistoreActive' => $this->get('prestashop.adapter.multistore_feature')->isActive(),
+            'isMultistoreActive' => $this->getMultistoreFeature()->isActive(),
```

Comment:
```suggestion
            'isMultistoreActive' => $this->getShopContext()->isMultiShopEnabled(),
```
Can you not use this now?